### ISSUE TITLE
[kahi_staff_udea_affiliations]

### ISSUE THREAD

**Issue by omazapa** [colav/Kahi_plugins#60]
```
Running plugin: kahi_staff_udea_affiliations
Traceback (most recent call last):
  File "/home/ozapatam/.local/bin/kahi_run", line 37, in <module>
    kahi.run()
  File "/home/ozapatam/.local/lib/python3.11/site-packages/kahi/Kahi.py", line 121, in run
    plugin_instance = plugin_class(config=plugin_config)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ozapatam/.local/lib/python3.11/site-packages/kahi_staff_udea_affiliations/Kahi_staff_udea_affiliations.py", line 36, in __init__
    raise Exception("University of Antioquia not found in database")
Exception: University of Antioquia not found in database

```

**Comment by muzgash** [comment]
To include anything from udea, ror and wikipedia plugins must finish successfully.

**Comment by omazapa** [comment]
then plugins must have dependency between them or give a message to the user about to run 
the required plugins first. 

I think is not bad to have some plugins that depend on other plugins in the workflow.

**Comment by muzgash** [comment]
Fixed in version 0.0.3a0
